omod/src/test/java/org/openmrs/Allergen.java

```diff
@@ -0,0 +1,4 @@
+package org.openmrs;
+
```

Comment:
Can you add some comments for why this class was added? Will be very useful to even yourself when you, after some years, ask yourself, why the hell you added this. 😊